app/src/main/java/com/github/se/signify/ui/navigation/BottomNavigationMenu.kt

```diff
@@ -29,12 +28,12 @@ fun BottomNavigationMenu(
   Box(
       modifier =
           modifier
-              .border(2.dp, colorResource(R.color.blue)) // Top blue line
+              .border(2.dp, MaterialTheme.colorScheme.outlineVariant) // Top blue line
               .padding(bottom = 2.dp)
               .testTag("BottomNavigationBar")) {
         NavigationBar(
             modifier = Modifier.fillMaxWidth().height(60.dp),
-            containerColor = colorResource(R.color.white) // Set background to white
+            containerColor = MaterialTheme.colorScheme.background // Set background to white
```

Comment:
Example of a possibly outdated comment as now the colors are theme dependent.

Reply:
For blatant issues like this one, don't hesitate to `Request changes`. I can always dismiss the request if I really don't want to listen anyway.